odra-macros/src/ast/deployer_item.rs

```diff
@@ -1,14 +1,12 @@
+#![allow(unused_variables)]
```

Comment:
Can this be removed?

Reply:
No, once the `validate` function is no longer required for `InitArgs`, the implementation is empty. Now, some derived `ToTokens` code causes a lint issue we need to ignore.